Fix a simple issue in latency_tracer.c

Fix a simple issue in latency_tracer.c

Signed-off-by: Jan Altenberg <jan@linutronix.de>
---
 kernel/latency_trace.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: linux-2.6.24-rt1/kernel/latency_trace.c
===================================================================
--- linux-2.6.24-rt1.orig/kernel/latency_trace.c	2008-01-25 15:07:15.000000000 -0500
+++ linux-2.6.24-rt1/kernel/latency_trace.c	2008-01-25 15:07:15.000000000 -0500
@@ -1017,7 +1017,7 @@ static int min_idx(struct block_idx *bid
 		idx = bidx->idx[cpu];
 		if (idx >= min(max_tr.traces[cpu].trace_idx, MAX_TRACE))
 			continue;
-		if (idx >= MAX_TRACE*NR_CPUS) {
+		if (idx > MAX_TRACE*NR_CPUS) {
 			printk("huh: idx (%d) > %ld*%d!\n", idx, MAX_TRACE,
 				NR_CPUS);
 			WARN_ON(1);
@@ -1164,7 +1164,7 @@ static void update_out_trace(void)
 		*out_entry = *entry;
 		out_entry++;
 		sum++;
-		if (sum >= MAX_TRACE*NR_CPUS) {
+		if (sum > MAX_TRACE*NR_CPUS) {
 			printk("huh: sum (%d) > %ld*%d!\n", sum, MAX_TRACE,
 				NR_CPUS);
 			WARN_ON(1);
